src/styles/main.scss

```diff
@@ -0,0 +1,12 @@
+@import './variables';
+
+body {
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  display: flex;
```

Comment:
The 'display: flex' property should be placed before 'flex-direction', 'justify-content', and 'align-items' properties to follow the logical sequence and improve readability as per the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules).